src/hclib_cpp.cpp

```diff
@@ -36,6 +36,26 @@ void hclib::get_places(hclib::place_t **pls, hclib::place_type_t type) {
     hclib_get_places(pls, type);
 }
 
+place_t *hclib::get_current_place() {
+    return hclib_get_current_place();
+}
+
+place_t **hclib::get_children_of_place(place_t *pl, int *num_children) {
+    return hclib_get_children_of_place(pl, num_children);
+}
+
+place_t *hclib::get_root_place() {
+    return hclib_get_root_place();
+}
+
+place_t **hclib::get_nvgpu_places(int *n_nvgpu_places) {
+    return hclib_get_nvgpu_places(n_nvgpu_places);
+}
+
+char *hclib::get_place_name(place_t *pl) {
+    return hclib_get_place_name(pl);
+}
+
 void *hclib::ddf_wait(hclib::ddf_t *ddf) {
```

Comment:
Do you mean you want to capitalize everything? See my comments above on why DDF_WAIT was originally capitalized, and why that's a bad idea. I don't think I can name a single API that uses capitalization as its style (except for maybe fortran or sequel stuff, certainly not C/C++ stuff).